04-shapes/Shapes.java

```diff
@@ -0,0 +1,31 @@
+import java.until.*;
+
+ public class Shapes{
+   public static void main(String[] args){
+     //Square//
+     double Side  = 5.8;
+
+     //Triangle//
+     double TriB = 10.6;
+     double TriH = 4.5;
+     //double Half = 0.5;
+
+     //variables holding the values//
+     double Squ = square(Side);
+     double Tri = triangle(TriB*TriH*0.5);
```

Comment:
does this actually run? doesn't triangle require 2 arguments passed in? i don't see a comma separating 2 arguments.